app/javascript/controllers/form_validation_controller.js

```diff
@@ -38,12 +46,14 @@ export default class FormValidation extends ValidationController {
     if (this.errors.has(attr)) {
       FormValidation.applyErrorStylesTo(el);
       FormValidation.errorMessageEl(el).textContent = this.errorMessage(attr);
+    } else {
+      FormValidation.removeError(el);
     }
   }
 
   static errorMessageEl(el) {
     const errorField = el.parentElement.nextElementSibling;
-    errorField.classList.remove('hidden');
+    errorField.classList.toggle('hidden');
```

Comment:
Was this change intentional @crespire?

Whiling QA'ing, I noticed the some odd behaviour due to the toggling:

https://user-images.githubusercontent.com/7963776/194119534-69f332c0-6b4c-4562-8aec-83a2ee7e2800.mov

Reply:
But it does work as expected when we use `remove` here instead as far as I can tell.

Reply:
Ah, in that case, I can change it back! I thought toggle might be a better approach. I did some light QA and ran the tests after changing to `toggle` but seems `add/remove` is a better approach.

Will revert that commit now.